### PR TITLE
Style/DateTime rubocop

### PR DESCRIPTION
**Pull request by drEnilight**
Hi. Started refactoring code which  ignore in `.rubocop_todo.yml`

### CODE REVIEW

File: lib/ossert/repositories.rb [hunk 1]
```diff
@@ -101,7 +101,7 @@ def initialize(stored_project, stats_type)
       end
 
       def coerce_value(value)
-        DateTime.parse(value)
+        Date.parse(value)
```
Comment (dsalahutdinov) [inline lib/ossert/repositories.rb]: Does it works fine with`Date` instead of `DateTime`?

Comment (drEnilight) [reply lib/ossert/repositories.rb]: yes, I checked.

Comment (dsalahutdinov) [reply lib/ossert/repositories.rb]: @sclinede

File: lib/ossert/fetch/github.rb [hunk 2]
```diff
@@ -123,7 +123,7 @@ def date_from_tag(sha)
         tag_info = tag_info(sha)
         return tag_info[:tagger][:date] if tag_info
         value = commit(sha)[:commit][:committer][:date]
-        DateTime.new(*value.split('-').map(&:to_i)).to_i
+        Date.new(*value.split('-').map(&:to_i)).to_time(:utc).to_i
```
Comment (sclinede) [inline lib/ossert/fetch/github.rb]: Sorry, but here I have some concerns.
I hope that some day we'll get rid of the ActiveSupport.
`#to_time(:utc)` is unfortunately - is a part of it...
Let's try to find other way to fix.
WDYT?

File: lib/ossert/repositories.rb (hunk 1 above)
Comment (sclinede) [reply lib/ossert/repositories.rb]: Could you, please, verify it without ActiveSupport?

File: lib/ossert/fetch/github.rb (hunk 2 above)
Comment (drEnilight) [reply lib/ossert/fetch/github.rb]: Ok, i can try.

**Comment by drEnilight** [comment]
@sclinede rewrite without ```#to_time(:utc)```. What do you think?

**Comment by sclinede** [comment]
@drEnilight Maybe we could introduce some Refinements for Dates ? And remove `active-support` dependency. WDYT?

**Comment by sclinede** [comment]
@drEnilight any progress? :)

**Comment by drEnilight** [comment]
@sclinede i'm sorry. But now, i haven't any time. :(

**Comment by drEnilight** [comment]
closed

**Pull request closed without merging**